src/components/claims/claim-details/components/claim-items/components/ItemForm.tsx

```diff
@@ -26,40 +32,59 @@ export const ItemForm: React.FC<{
   const [dateOfPurchase, setDateOfPurchase] = React.useState('')
   const [note, setNote] = React.useState('')
   const [purchasePriceCurrency, setPurchasePriceCurrency] = React.useState('')
-
+  const [autoValuation, setAutoValuation] = React.useState<string | null>('')
+  const [customValuation, setCustomValuation] = React.useState('')
   const [contractMarketInfo] = useContractMarketInfo(memberId ?? '')
-  const validPurchasePrice = /^[0-9]*$/g.test(purchasePrice)
   const defaultCurrency = contractMarketInfo?.preferredCurrency ?? 'SEK'
 
   const [
     upsertClaimItem,
     { loading: loadingUpsertClaimItem },
   ] = useUpsertClaimItemMutation()
 
-  const addNewClaimItemRequest: UpsertClaimItemInput = {
+  const formData: UpsertClaimItemInput = {
```

Comment:
Since this is used for the request only I would name it after the variable, either:
```suggestion
  const request: UpsertClaimItemInput = {
```
Or
```suggestion
  const input: UpsertClaimItemInput = {
```